src/terra-dev-site/guide/Internationalization/TestingComponentsWhichIncludeTranslations.f.guide.md

```diff
@@ -39,4 +27,30 @@ describe('<Toggle />', () => {
 });
 ```
 
+## Aggregating translations for Jest tests
+With [terra-enzyme-intl](https://github.com/cerner/terra-enzyme-intl), aggregating translations for Jest tests is not necessary. The terra-enzyme-intl package will use the translation keys for the translation strings  instead of the actual translations. If you want to test the actual translation string values, we recommend testing translations with a tool like webdriver.io.
```

Comment:
extra space
```suggestion
With [terra-enzyme-intl](https://github.com/cerner/terra-enzyme-intl), aggregating translations for Jest tests is not necessary. The terra-enzyme-intl package will use the translation keys for the translation strings instead of the actual translations. If you want to test the actual translation string values, we recommend testing translations with a tool like webdriver.io.
```

Reply:
👍